### DISCUSSION TITLE
统计时间出了点问题。

### DISCUSSION THREAD

**Discussion by rebron1900** [Planshit/Tai#11] [Q&A] [answered]
统计时间似乎有点问题，可能是因为我的屏幕关闭了，但是电脑没关闭，导致记录了一整夜的使用时长。
![image](https://user-images.githubusercontent.com/7929884/146871104-c09f40e5-8ade-4ed5-aa7b-6c8e634437de.png)

**Comment by rebron1900** [comment]
这种情况请问如何处理。

**Comment by noberumotto** [comment]
谢谢反馈，应该是程序bug，下一个版本将修复。麻烦将data\data.db文件上传一下。

**Comment by rebron1900** [comment]
[Tai.zip](https://github.com/Planshit/Tai/files/7766005/Tai.zip)
@noberumotto

**Comment by rebron1900** [comment, marked as answer]
我自己把那条数据删掉了。

**Comment by noberumotto** [comment]
我想问下你是否还记得出现这个问题的时候，当天或者前一天有没有使用其他设备以远程桌面连接的方式登录过这台电脑？

**Reply by rebron1900** [reply]
没有，我估计是我关了显示器，但是我的电脑没关，但是又没锁屏。
可能是这么个情况。我觉得要加个判断，如果长时间没使用键盘鼠标可以考虑把这部分数据列入异常计数。交给用户来判定是否计入统计。

**Reply by noberumotto** [reply]
这个情况在开发之初已经考虑到并做了处理了，而且最新的1.0.0.1版本对判断是否离开电脑做了更加全面的处理。

我也复现了统计时间异常这个问题，但不是像你这个情况发生的，我是在远程桌面连接断开后离开了电脑，再次打开时统计时间才出现问题。

还有我想问一下现在你是否还会碰到这个统计时间异常的Bug呢？

**Reply by rebron1900** [reply]
昨天又碰到了，看进程名称好像是文件选择窗口
